site/server/app/com/fortysevendeg/exercises/services/free/userops.scala

```diff
@@ -0,0 +1,30 @@
+package com.fortysevendeg.exercises.services.free
+
+import scala.language.higherKinds
+
+import cats.data.Xor
+import cats.free.Free
+import cats.free.Inject
+import shared.ExerciseEvaluation
+import shared.Library
+import shared.Section
+
+/** Exercise Ops GADT
```

Comment:
There are a few mentions to `Exercise` instead of `User` in this file, probably because copy-pasting the free boilerplate.